charts/ai-software-templates/chatbot/values.yaml

```diff
@@ -14,6 +14,8 @@ gitops:
   gitSecretName: "github-secrets"
   gitSecretKeyToken: "GITHUB_TOKEN"
   gitSourceRepo: "redhat-ai-dev/ai-lab-samples"
+  gitAuthorName: "Your Name"
```

Comment:
@gabemontero In case they are required might be better if we leave them empty? wdyt?

Btw, one thing I'd like to add for the next version of the helm chart is to not allow installing if any of `githubOrgName`, `quayAccountName` are empty (we now only highlight that they are required in the readme and json schema)

Reply:
per @maysunfaisal 's comment thread @thepetk I've figured out how to set this without requiring this parameter.

this and gitAuthorEmail will be removed in my next push